CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This is a changelog for Piwik platform developers. All changes for our HTTP API's, Plugins, Themes, etc will be listed here.
 
+## Piwik 3.0.0
+
+### Breaking Changes
```

Comment:
@sgiehl  Can you also note in CHANGELOG that `prettyDate` attribute on API response has slightly changed?